index-html/about.html

```diff
@@ -0,0 +1,152 @@
+        </a>
+        <a href="#" class="top-nav-link">
+          <i class="fa-brands fa-twitter"></i>
+        </a>
+        <a href="#" class="top-nav-link"> English </a>
+        <a href="#" class="top-nav-link"> myPage </a>
+        <a href="#" class="top-nav-link last"> logout </a>
+      </div>
+      <nav class="desktop">
+        <div class="desktop-logo">
+          <a href="index.html" class="desk-logo">
+            <img src="images/logo.png" alt="logo" />
+          </a>
+        </div>
+        <div class="desktop-nav">
+          <ul class="desktop-nav-lists">
+            <li class="nav-list">
+              <a class="nav-link" href="about.html">About</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="#program">Program</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="index.html">Speakers</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="index.html#Sponsors">Sponsor</a>
+            </li>
+            <li class="nav-list">
+              <a href="">
+                <button class="nav-btn btn">CC Campaign</button>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <!-- Hero section -->
+    <section class="hero-about" id="home">
+      <div class="nav-toggle">
+        <div class="burger bar1"></div>
+        <div class="burger bar2"></div>
+        <div class="burger bar3"></div>
+      </div>
+      <nav class="mobile-nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-item">
+            <a href="about.html" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#program" class="nav-link">Program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#speakers" class="nav-link">Speakers</a>
+          </li>
+        </ul>
+        <a href="">
+          <button class="nav-btn btn">CC Campaign</button>
+        </a>
+      </nav>
+      <div class="hero-body">
+        <img
+          class="about-bg-img"
+          src="./images/about_title.png"
+          alt="about hero background"
+        />
+        <div class="about-hero-underline"></div>
+        <p class="hero-subtitle">
+          The Education Enclave, organized biennially, serves as a hub where
+          advocates of open education from diverse backgrounds, including
+          educators, researchers, and activists, converge. This event brings
+          together individuals committed to fostering open culture and those who
+          actively promote and participate in the field of education.
+        </p>
+        <p class="contact">
+          Please contact us via email for any further question about CC Global
+          Submit 2015! <br /><a href="#">ccsummit@gmail.com </a>
```

Comment:
- [ ] kindly consider giving this email link the same color as the one in the given design to implement exactly what expected from the given design 😃

Reply:
Done